### PR TITLE
Add bnet database authentication

### PR DESCRIPTION
**Pull request by quoing**

### CODE REVIEW

File: scripts/create_default_user.py [hunk 1]
```diff
@@ -70,6 +71,21 @@ def main():
 	connection.commit()
 	connection.close()
 
+        # create bnet.db with default account (based on stove/wiki)
+	connection_bnet = sqlite3.connect(dbfile_bnet)
+        cursor_bnet = connection_bnet.cursor()
```
Comment (Mischanix) [inline scripts/create_default_user.py]: don't mix spaces/tabs

**Comment by quoing** [comment]
Indenting corrected. I wonder why python didn't complain about it while I was running tests ;)

**Comment by jleclanche** [comment]
You're using Python 2 I guess, because Python 3 would crash. You're still mixing spaces and tabs, btw.

**Comment by quoing** [comment]
Yup. Correcting that one also. 
And added bnet.db rewrite protection for bootstrap.

**Comment by Mischanix** [comment]
LGTM; in master. You'll need to set your pull request branch to d9b442e for github to consider the PR merged.

**Comment by jleclanche** [comment]
No need to set the branch, just rebase on top of master :)

**Pull request closed without merging**